AdtSampleApp/SampleFunctionsApp/local.settings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "IsEncrypted": "false",
+  "AdtConfiguration": {
+    "InstanceUrl": ""
+  },
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
```

Comment:
One way to further simplify the dependencies of this sample would be to remove using Triggers that require external services, such as EventGrid or IotHub. If we were to use just a TimerTrigger, no storage account is needed altogether. If we do want to keep the IotHub trigger however, then we need to tell people to launch the Azure SDK Storage Emulator if they want to debug the function on their dev box.